pages/doc/proxies_versions.md

```diff
@@ -18,6 +18,9 @@ This page gives an overview of important changes for the most recent Wavefront p
 - [Jaeger integration can now receive data via HTTP](proxies_configuring.html#traceJaegerHttpListenerPorts).
 - Log blocked points for [histograms and spans into separate log files](proxies_configuring.html#logging).
 - [Ability to export data that is queued at the proxy](proxies_installing.html#export-data-queued-at-the-proxy).
+- Deprecated configurations:
```

Comment:
Are there really only 2?

Reply:
Fingers crossed, yes! :) Will get it reviewed.